skate-plugin/project-gen/src/jvmMain/kotlin/slack/tooling/aibot/ChatWindow.kt

```diff
@@ -34,30 +39,44 @@ import androidx.compose.ui.input.key.onPreviewKeyEvent
 import androidx.compose.ui.input.key.type
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.text.TextRange
+import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.text.input.TextFieldValue
 import androidx.compose.ui.unit.dp
 import org.jetbrains.jewel.foundation.theme.JewelTheme
-import org.jetbrains.jewel.ui.component.Icon
-import org.jetbrains.jewel.ui.component.IconButton
-import org.jetbrains.jewel.ui.component.Text
-import org.jetbrains.jewel.ui.component.TextArea
+import org.jetbrains.jewel.ui.component.*
 
 @Composable
 fun ChatWindow(modifier: Modifier = Modifier) {
```

Comment:
We're getting a little to the point now logic-wise that I think it would be wise to start using Circuit for this. Doesn't have to be in this PR, but I think it's probably worth it for the next PR